test/presenters/service_manual_homepage_presenter_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class ServiceManualHomepagePresenterTest < ActiveSupport::TestCase
+  test '#topics returns the children in the links, ordered alphabetically' do
+    homepage = presented_homepage(
+      'links' => {
+        'children' => [
+          { 'title' => "Agile Delivery", 'base_path' => "/service-manual/agile-delivery", 'description' => "Do this." },
+          { 'title' => "Helping people to use your service", 'base_path' => "/service-manual/helping-people-to-use-your-service", 'description' => "And this" },
+          { 'title' => "Funding and procurement", 'base_path' => "/service-manual/funding-and-procurement", 'description' => "This too." }
+        ]
+      }
+    )
+
+    assert_equal(
+      [
+        { 'title' => "Agile Delivery", 'base_path' => "/service-manual/agile-delivery", 'description' => "Do this." },
+        { 'title' => "Funding and procurement", 'base_path' => "/service-manual/funding-and-procurement", 'description' => "This too." },
+        { 'title' => "Helping people to use your service", 'base_path' => "/service-manual/helping-people-to-use-your-service", 'description' => "And this" }
+      ],
+      homepage.topics
+    )
+  end
```

Comment:
I think I would have just included the `title` in these fixtures, instead of `title`, `base_path` and `description`. It would make them a bit quicker to grok and it's all the data we need for this unit test. Up to you.